spatialpandas/tests/test_parquet_s3.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import os
+import shlex
+import subprocess
+import sys
+import time
+
+import dask.dataframe as dd
+import numpy as np
+import pytest
+
+from spatialpandas import GeoDataFrame, GeoSeries
+from spatialpandas.geometry import PointArray
+from spatialpandas.io import read_parquet, read_parquet_dask, to_parquet, to_parquet_dask
+
+
+pytest.importorskip("moto")
+geopandas = pytest.importorskip("geopandas")
+s3fs = pytest.importorskip("s3fs")
+requests = pytest.importorskip("requests")
+
+if sys.version_info < (3, 8):
+    pytest.skip("requires python3.8 or higher", allow_module_level=True)
+
+logging.getLogger("botocore").setLevel(logging.INFO)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def s3_fixture():
+    """Writable local S3 system."""
```

Comment:
I think it would be useful to add a comment here saying where this is taken from in `s3fs`.

Reply:
Added, although this was borrowed circuitously through universal_pathlib, so I left a comment attributing both.

Reply:
Thanks.